src/maintenance/setup.py

```diff
@@ -1,40 +1,57 @@
+#!/usr/bin/env python
+
 # --------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
 
+from codecs import open
 from setuptools import setup, find_packages
 
+# HISTORY.rst entry.
+VERSION = '0.1.0'
+try:
+    from azext_maintenance.manual.version import VERSION
+except ImportError:
+    pass
 
-VERSION = "1.0.1"
```

Comment:
If we have had 1.0.1 released, I think we need a bigger version number:
https://github.com/Azure/azure-cli-extensions/pull/2214/files#diff-ba12df623efca123165ca48252645955R13
https://github.com/Azure/azure-cli-extensions/pull/2214/files#diff-2503d1af174da5a5c12c1c6542850ce9R6